src/storage/v2/indices/indices.cpp

```diff
@@ -42,6 +42,11 @@ void Indices::RemoveObsoleteEntries(uint64_t oldest_active_start_timestamp, std:
       ->RemoveObsoleteEntries(oldest_active_start_timestamp, std::move(token));
 }
 
+void Indices::DropGraphClearIndices() const {
+  static_cast<InMemoryLabelIndex *>(label_index_.get())->DropGraphClearIndices();
+  static_cast<InMemoryLabelPropertyIndex *>(label_property_index_.get())->DropGraphClearIndices();
+}
```

Comment:
Since we added `EdgeTypeIndices`, I think dropping them should also be here.

Reply:
[InMemoryEdgeTypeIndex](https://github.com/memgraph/memgraph/blob/master/src/storage/v2/inmemory/edge_type_index.hpp)